internal/core/items.rs

```diff
@@ -540,25 +513,15 @@ impl Item for TouchArea {
                 InputEventResult::EventAccepted
             }
 
-            MouseEvent::Released { button, position, .. } => {
+            MouseEvent::Released { button, position, click_count } => {
                 let geometry = self_rc.geometry();
-                let double_clicked_set =
-                    Self::FIELD_OFFSETS.double_clicked.apply_pin(self).has_handler();
-
                 if button == PointerEventButton::Left
                     && LogicalRect::new(LogicalPoint::default(), geometry.size).contains(position)
                     && self.pressed()
                 {
                     Self::FIELD_OFFSETS.clicked.apply_pin(self).call(&());
-                    if double_clicked_set {
-                        let in_double_click_interval = self.had_double_click();
-                        self.reset_last_click_timestamp();
-
-                        if in_double_click_interval {
-                            Self::FIELD_OFFSETS.double_clicked.apply_pin(self).call(&())
-                        } else {
-                            self.touch_last_click_timestamp();
-                        }
+                    if (click_count % 2) == 1 {
```

Comment:
I'm always a bit confused by the click_count ... counting. Looking forward to documenting this :)

Reply:
this was one of the reason we didn't stabilize it yet.
But it starts at 0.